src/components/Collapse/index.tsx

```diff
@@ -50,7 +50,7 @@ const Collapse: React.FC<CollapseProps> = ({
 
   return (
     <Wrapper>
-      <HeaderWrapper onClick={handleClick}>
+      <HeaderWrapper tabIndex={0} role="button" aria-pressed="false" onClick={handleClick}>
```

Comment:
I love this point. However, this is not as simple as just replacing it from div to button due to styling and the element children already there because we'd have to make sure that the structure is valid.

For the moment, I took the suggestion for the key down event